### ISSUE TITLE
Incompatilble with your PHP Version

### ISSUE THREAD

**Issue by mageuser** [firebearstudio/importexportfree#24]
[InvalidArgumentException]
  Package firebear/importexportfree at version  has a PHP requirement incompatible with your PHP
  version (7.1.9)

i've checked composer.json
` "php": "~5.5.0|~5.6.0|~7.0.0|~7.1",`
how solve this?

**Comment by maryxyan** [comment]
Yeah ... same problem, can someone help us please?
  [InvalidArgumentException]
  Could not find package firebear/importexportfree at any version matching your PHP version 7.1.14.0

**Comment by asunyer90** [comment]
Hi! Same issue here!

**Comment by amenk** [comment]
strange ... shouldn't it work? https://jubianchi.github.io/~7.1/7.1.14

**Comment by amenk** [comment]
wait, which version of importexportfree are you trying to install @kauansoft ? maybe it's picking up an old version which did not have yet the requirement. did you try to require `firebear/importexportfree=dev-master`

**Comment by magenizr** [comment]
FYI. We tried to install v1.0.3 twice on a clean Magento CE 2.3.0 + Sample Data / PHP 7.1.27 but always got the below exception.

```
[InvalidArgumentException]
Package firebear/importexportfree at version v1.0.3 has a PHP requirement incompatible with your PHP version (7.1.27)
```

As @amenk has mentioned, `dev-master` is working fine.

**Comment by biotech** [comment]
@magenizr @kauansoft thank you for feedback , we will test and merge to master asap

**Comment by nikolaevas** [comment]
@magenizr Hi. Did you try to install the latest version v1.0.13? Because version 1.0.3 is old and doesn't compatible with php 7.1.

**Comment by magenizr** [comment]
@nikolaevas @biotech You're right. I mixed up 1.0.3 with 1.0.13 somehow, sorry about that. I confirm that 1.0.13 is working fine.

```
- Installing firebear/importexportfree (1.0.13): Cloning ecb066abff from cache
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.
Writing lock file
Generating autoload files
```
